kosat-core/src/commonMain/kotlin/org/kosat/CDCL.kt

```diff
@@ -416,7 +416,14 @@ class CDCL {
 
     // ---- CDCL functions ---- //
 
-    // add new constraint and watchers to it, executes only in newClause
+    /**
+     * Add new constraint and watchers to it.
+     *
+     * This function assumes that the clause size is at least 2,
+     * and it is expected to be run by the solver internally to
+     * add constraints to the solver. This will be moved to the
+     * proper clause database in the future.
+     */
     private fun addClause(clause: Clause) {
         require(clause.size != 1)
```

Comment:
Be consistent: either use `clause.size != 1` or `clause.size > 1` everywhere. I believe Clause already guarantees `clause.size > 0`, so these two variant are equivalent, but nevertheless, try to be consistent.